ChannelMap/ChannelMap.cxx

```diff
@@ -48,6 +48,17 @@ namespace emph {
 
     //----------------------------------------------------------------------
 
+    void ChannelMap::PrintE2D()
+    {
+      std::map<emph::cmap::EChannel,emph::cmap::DChannel>::iterator itr=fEChanMap.begin();
+      
+      for (; itr != fEChanMap.end(); ++itr) {
+	std::cout << itr->first << "-->" << itr->second << std::endl;
```

Comment:
we should probably replace cout's with mf::LogInfo etc, so we can control the level of printouts in fcls. not worth holding up this PR though atm

Reply:
Ok, I'll put that on my to-do list.  We should probably do this throughout our code...